docs/sagemaker/train.md

```diff
@@ -99,6 +99,12 @@ _Note that SageMaker doesn’t support argparse actions. For example, if you wan
 
 Look [here](https://github.com/huggingface/notebooks/blob/master/sagemaker/01_getting_started_pytorch/scripts/train.py) for a complete example of a 🤗 Transformers training script.
 
+## Training Output Management
+
+`output_dir` defaults to '/opt/ml/model' which saves both checkpoints and models to '/opt/ml/model', and results in a large output `tar.gz` containing all checkpoint files and model. A large artifacts most of them possibly useless for deployment.
+
+If you want to persist checkpoints to Amazon S3 while saving only the final model in the `model.tar.gz` final training artifact, you can point Hugging Face Trainer’s `output_dir` to `/opt/ml/checkpoints` and at the end of your script save a specific model to the model location with `trainer.save_model(/opt/ml/model)`
```

Comment:
Don't you need to provide [...] in the Estimator to enable checkpointing?
```bash
    # Parameters required to enable checkpointing
    checkpoint_s3_uri=checkpoint_s3_bucket,
    checkpoint_local_path=checkpoint_local_path
``` 
 [Enable Checkpointing documentation](https://docs.aws.amazon.com/sagemaker/latest/dg/model-checkpoints.html)

Reply:
Maybe this can be added to the section?

Reply:
What do you mean by "this"? 
If it is right that the user needs to make more changes than just saving `/opt/ml/checkpoints` we need to include these changes as well.

Reply:
I meant clarifying how to separately save both checkpoints and models to S3. It will be make the section clearer how to save checkpoints by adding the code snippet above in your comment.

Reply:
Then let's remove it from this section and add an additional one?

Reply:
It is not necessary to create another section, it can be in same section and link the checkpointing documentation.

Reply:
Ok👍🏻 i ll wait for your changes and then take another look.

Reply:
I have updated the checkpointing part.